www/source/blog/2017-10-05-Introducing-Composites.html.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Introducing Composites
+date: 2017-10-05
+author: Christopher Maier
+tags: blog
```

Comment:
You might want to copy the categories into this field, too, or it'll just be tagged as "blog" to the left pane of the blog